pkg/operator/controllers/machinehealthcheck/staticresources/mhcremediationalert.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: mhc-remediation-alert
+  namespace: openshift-machine-api
```

Comment:
Can we also add the best-practice labeling structure? Setting the Prometheus instance is of particular concern - if customers create their own Prometheus, these may fire on thier instance instead of the cluster instance.

ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/alerting.md#prometheusrule-labelling

Reply:
FYI:

I just happened to have a new cluster spun up, looks like we'd need to set this to `prometheus: k8s` to accommodate the OpenShift defaults:

```sh
$ oc get Prometheus -A
NAMESPACE              NAME   VERSION   REPLICAS   AGE
openshift-monitoring   k8s    2.29.2    2          16m
```

Reply:
Yes, I will add this and update shortly. Thank you for pointing this out!